octoprint_discordremote/discord.py

```diff
@@ -319,47 +336,58 @@ def handle_heartbeat_ack(self):
             self.status_callback(connected="connected")
         if self.error_counter > 0:
             self.error_counter -= 0
-        self.heartbeat_sent = 0
+        self.heartbeat_sent=0
         self.process_queue()
 
     def handle_ready(self, js):
         self.logger.info("Received READY message")
-        self.last_sequence = js['s']
-        self.session_id = js['d']['session_id']
+        self.last_sequence=js['s']
+        self.session_id=js['d']['session_id']
 
     def handle_invalid_session(self, js):
         self.logger.info("Received INVALID_SESSION message")
-        self.session_id = None
-        self.last_sequence = None
+        self.session_id=None
+        self.last_sequence=None
         time.sleep(5)
         self.send_identify()
 
     def process_queue(self):
         while not self.shutdown_event.is_set() and len(self.queue):
-            snapshot, embed = self.queue.pop()
+            snapshot, embed=self.queue.pop()
             if self._dispatch_message(snapshot=snapshot, embed=embed):
                 continue
             else:
                 break
 
-    def send(self, snapshots=None, embeds=None):
+    def send(self, channel_id=None, snapshots=None, embeds=None):
         if snapshots is not None:
             for snapshot in snapshots:
-                if not self._dispatch_message(snapshot=snapshot):
-                    return False
+                if channel_id == None:
+                    for f in self.channels_id.split(','):
```

Comment:
- [ ] Use a more descriptive variable
```suggestion
                    for channel_id in self.channels_id.split(','):
```

- [ ] Consider splitting on any non-numerical value, so that if a user has "123561324; 1234152123,       28437569345", it will still parse correctly.

Reply:
Hi !
Do you mean I should automaticaly remove the space symbol ?

Reply:
Yes, i cant find it, but i had some code that split on any non-numerical character, so that whats left is just a list of numbers